Packs/CarbonBlackDefense/Integrations/CarbonBlackLiveResponseCloud/CarbonBlackLiveResponseCloud.py

```diff
@@ -0,0 +1,313 @@
+
+    human_readable = [dict(name=val['value_name'], type=val['value_type'], data=val['value_data']) for val in values]
+
+    readable_output = tableToMarkdown('Carbon Black Defense Live Response Registry key values',
+                                      human_readable,
+                                      headers=['name', 'type', 'data'],
+                                      headerTransform=string_to_table_header,
+                                      removeNull=True)
+
+    return CommandResults(
+        outputs_prefix='CarbonBlackDefenseLR.RegistryValues',
+        outputs_key_field='sensor_id',
+        outputs=context_entry,
+        readable_output=readable_output,
+        raw_response=values
+    )
+
+
+# Using Py API
+def delete_reg_value_command(api_client: CBCloudAPI, sensor_id: str, reg_path: str):
+    session = api_client.select(endpoint_standard.Device, sensor_id).lr_session()
+    session.delete_registry_value(reg_path)
+    return f'Registry value: {reg_path} was deleted successfully.'
+
+
+# Using Py API
+def delete_reg_key_command(api_client: CBCloudAPI, sensor_id: str, reg_path: str):
+    """Delete a registry key on the remote machine, the key must be without any sub keys"""
+
+    session = api_client.select(endpoint_standard.Device, sensor_id).lr_session()
+    session.delete_registry_key(reg_path)
+    return f'Registry key: {reg_path} was deleted successfully.'
+
+
+# Using Py API
+def list_processes_command(api_client: CBCloudAPI, sensor_id: str):
+    session = api_client.select(endpoint_standard.Device, sensor_id).lr_session()
+    processes = session.list_processes()
+    if not processes:
+        return 'There is no active processes in the remote sensor'
+
+    headers = ['path', 'pid', 'command_line', 'username']
+    processes_readable = [dict(
+        path=process['path'],
+        pid=process['pid'],
+        command_line=process['command_line'],
+        user_name=process['username']) for process in processes]
+    context_entry = dict(sensor_id=sensor_id, processes=processes)
+
+    readable_output = tableToMarkdown('Carbon Black Defense Live Response Processes',
+                                      headers=headers,
+                                      t=processes_readable,
+                                      headerTransform=string_to_table_header,
+                                      removeNull=True)
+    return CommandResults(
+        'CarbonBlackDefenseLR.Processes',
+        outputs_key_field='sensor_id',
+        outputs=context_entry,
+        readable_output=readable_output,
+        raw_response=processes,
+    )
+
+
+# Using Py API
+def kill_process_command(api_client: CBCloudAPI, sensor_id: str, pid: Any):
+    session = api_client.select(endpoint_standard.Device, sensor_id).lr_session()
+    success = session.kill_process(pid)  # the API returns True if success, False if failure
+    if not success:
+        return_error(f'Can not kill the process: {pid}')
```

Comment:
The API didn't indicating why the return are False
But from inside the API code it's look that False is returned in case Of TimeoutError
Should i reflect this reason to the user (as we doesn't have ability to configure the timeout argument)